src/client/search/searchReducer.spec.js

```diff
@@ -23,7 +23,7 @@ describe('#SearchTermsReducer', () => {
       query: 'Football',
       paramTypes: createTwitterParamTypes(['hashtag']),
       source: 'twitter',
-    }];
+    }, ];
```

Comment:
This is the standard now, this won't change with each pull request.
Let me know if this messes with your ESLint